### PR TITLE
Hide stdout from shell commands

### PR DESCRIPTION
**Pull request by lukaszo**
Fix permissions for capact-local-ca.crt file on Linux systems

## Description

Changes proposed in this pull request:

- Hide stdout brom shell commands
- Fix permissions for capact-local-ca.crt file on Linux systems
- 
## Related issue(s)

#328

### CODE REVIEW

**Review by mszostok — APPROVED** [review]
still works on mac 👍

File: internal/cli/capact/os.go [hunk 1]
```diff
@@ -69,7 +61,7 @@ func TrustSelfSigned(status *printer.Status) error {
 
 func trustSelfSignedDarwin(tmpCertPath string) error {
 	// #nosec G204
-	cmd := exec.Command("/bin/sh", "-c", fmt.Sprintf("security verify-cert -c %s", tmpCertPath))
+	cmd := exec.Command("/bin/sh", "-c", fmt.Sprintf("security verify-cert -c %s >/dev/null", tmpCertPath))
 	cmd.Stdout = os.Stdout
```
Comment (mszostok) [inline internal/cli/capact/os.go:65]: isn't it easier to use io.Discard instead of having `/dev/null` in command?

**Comment by mszostok** [comment]
it still doesn't look good, but it's acceptable:
```
 ✓ Installing capact Helm chart
 ⣯ Trusting self-signed CA certificate if not already trusted Cert Verify Result: CSSMERR_TP_NOT_TRUSTED
 ⣾ Trusting self-signed CA certificate if not already trusted
 ✓ Trusting self-signed CA certificate if not already trusted
```

File: internal/cli/capact/os.go (hunk 1 above)
Comment (lukaszo) [reply internal/cli/capact/os.go:66]: ah, in this case yes. In cases when I'm using sudo no

**Pull request merged**